environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -257,9 +257,9 @@
     - name: Make cirros image visible
       ansible.builtin.command: openstack --os-cloud admin image set --public cirros  # noqa 301
 
-    - name: Download ubuntu minimal 20.04 image
+    - name: Download ubuntu minimal 22.04 image
       ansible.builtin.get_url:
-        url: http://cloud-images.ubuntu.com/minimal/releases/focal/release/ubuntu-20.04-minimal-cloudimg-amd64.img
+        url: http://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img
```

Comment:
Switch to using https?

Also make the URL a variable to avoid the duplication below?